tests/visco_plastic_yield_plastic_viscous_strain_healing.prm

```diff
@@ -0,0 +1,139 @@
+# Global parameters
```

Comment:
Since the reference Adiabatic surface temperature is not prescribed below, you are dividing by zero in the strain healing equation.